src/proxy-ui-api/src/main/resources/openapi-definition.yaml

```diff
@@ -4583,6 +4591,27 @@ components:
           uniqueItems: true
           items:
             $ref: '#/components/schemas/GroupMember'
+    LocalGroupAdd: # ok
+      type: object
+      description: model for adding a new LocalGroup
+      required:
+        - code
+        - description
```

Comment:
Is description actually required? Just wondering it doesn't sound mandatory.

Reply:
It is required. It is also shown as the main identifier when e.g. searching for subjects (in the first column in the subjects table) -> guess it would be dull if it was just empty